pkg/csv/http_writer.go

```diff
@@ -0,0 +1,93 @@
+package csv
+
+import (
+	"encoding/csv"
+	"fmt"
+	"net/http"
+
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/pkg/logging"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+const (
+	// CSVContentType is the Content-Type HTTP header value.
+	CSVContentType = "text/csv; charset=utf-8"
+	// UTF8BOM is the UTF-8 BOM byte sequence.
+	UTF8BOM = "\uFEFF"
+)
+
```

Comment:
Define a row type, and use it?
```suggestion
type Row []string
```